src/test/kotlin/lotto/domain/LottoNumberTest.kt

```diff
@@ -0,0 +1,23 @@
+package lotto.domain
+
+import io.kotest.assertions.throwables.shouldNotThrowAny
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+
+class LottoNumberTest : StringSpec() {
+    init {
+        "로또 넘버는 1~45만 가질 수 있다" {
+            shouldNotThrowAny {
+                for (i in 1..45) {
+                    LottoNumber.from(i)
+                }
+            }
+        }
+
+        "그 외의 숫자에 대해서 예외가 발생한다" {
+            shouldThrow<IllegalArgumentException> {
+                LottoNumber.from(0)
+            }
+        }
+    }
+}
```

Comment:
```suggestion
class LottoNumbersTest : StringSpec({
    "중복되지 않는 6개의 숫자를 리스트로 반환한다" {
        val numbers = LottoNumbers.getNumbers()

        numbers shouldHaveSize 6
        numbers shouldBe numbers.distinct()
    }
})
```

StringSpec안에 {} 을 만들면, init 없이 사용할 수 있습니다!